azure/services/managedclusters/spec.go

```diff
@@ -724,6 +731,10 @@ func adminKubeconfigSecretName(clusterName string) string {
 	return secret.Name(clusterName+"-aso", secret.Kubeconfig)
 }
 
+func oidcIssuerURLConfigMapName(clusterName string) string {
```

Comment:
let's set up a simple UT for this new func

Reply:
added via fe9c588ee0b05ef1f32b897f23b532fe76994260